sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Blobs/src/Listeners/ScanBlobScanLogHybridPollingStrategy.cs

```diff
@@ -260,7 +260,7 @@ public async Task<IEnumerable<BlobBaseClient>> PollNewBlobsAsync(
                 cancellationToken.ThrowIfCancellationRequested();
 
                 var properties = currentBlob.Properties;
-                DateTime lastModifiedTimestamp = properties.LastModified.Value.UtcDateTime;
+                DateTime lastModifiedTimestamp = properties.CreatedOn.Value.UtcDateTime;
```

Comment:
But won't this now not capture modification timestamps to existing blobs? The point of this optimization was to keep track of the latest timestamp we saw, so we can avoid re-inspecting blobs.

Reply:
It's been a long time since I've thought through this logic, but I thought that we knew this wasn't perfect but that when combined with the log scanning (PollLogsStrategy), we would fill the gaps eventually.